.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to `thin-edge.io`?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```

Comment:
I would add a 4th entry for breaking changes.

Reply:
Yep good, i added this point